Source/RunActivity/Viewer3D/Cameras.cs

```diff
@@ -1076,7 +1080,7 @@ protected override void OnActivate(bool sameCamera)
             }
             else
             {
-                isVisibleTrainCarViewerOrWebpage = Viewer.TrainCarOperationsViewerWindow.Visible || (Viewer.TrainCarOperationsWebpage.Connections > 0 && Viewer.TrainCarOperationsWebpage.TrainCarSelected);
+                isVisibleTrainCarViewerOrWebpage = (Viewer.TrainCarOperationsWindow.Visible && !Viewer.TrainCarOperationsViewerWindow.Visible) || Viewer.TrainCarOperationsViewerWindow.Visible || (Viewer.TrainCarOperationsWebpage?.Connections > 0 && Viewer.TrainCarOperationsWebpage.TrainCarSelected);
```

Comment:
The && and the first || is interesting here, I don't think the expression is correct here. Could you please double check? Also the ? mark  and the same object after & & without ? is incorrect